show/chassis_modules.py

```diff
@@ -102,3 +108,55 @@ def midplane_status(chassis_module_name):
         click.echo(tabulate(table, header, tablefmt='simple', stralign='right'))
     else:
         click.echo('No data available in CHASSIS_MIDPLANE_TABLE\n')
+
+@chassis.command()
+@click.argument('systemportname', required=False)
+@click.option('--namespace', '-n', 'namespace', required=True if multi_asic.is_multi_asic() else False, 
+                default=None, type=str, show_default=False, help='Namespace name or all')
+@click.option('--verbose', is_flag=True, help="Enable verbose output")
+def system_ports(systemportname, namespace, verbose):
+    """Show VOQ system ports information"""
+
+    cmd = "voqutil -c system_ports"
+
+    if systemportname is not None:
+        cmd += " -i \"{}\"".format(systemportname)
+    
+    if namespace is not None:
+        cmd += " -n {}".format(namespace)
+
+    clicommon.run_command(cmd, display_cmd=verbose)
+
+@chassis.command()
+@click.argument('ipaddress', required=False)
+@click.option('--asicname', '-x', 'asicname', default=None, type=str, show_default=False, help='Asic name')
+@click.option('--verbose', is_flag=True, help="Enable verbose output")
+def system_neighbors(asicname, ipaddress, verbose):
+    """Show VOQ system neighbors information"""
+
+    cmd = "voqutil -c system_neighbors"
+
+    if ipaddress is not None:
+        cmd += " -a {}".format(ipaddress)
+
+    if asicname is not None:
```

Comment:
@vganesan-nokia this change will not work on the single asic linecards right ?
I think it will be better to the add the asicname check only on multi asic linecards ?

Reply:
Please note that for system neighbors and system lags this is not namespace. It is the value of DEVICE_METADATA['localhost']['asic_name'] we filter on in keys .  Therefore, this will/should work on single asic linecards also for the following reasons: (1) Even for the single asic line card, in voq chassis, it is mandatory to configure DEVICE_METADATA['localhost']['asic_name'].  For single asic linecards, it may be always "asic0" or "Asic0". (2) Even if the command is executed from single asic linecard, we should be able to filter the entries for asic name of other line cards that have multiple asics.